django/views/debug.py

```diff
@@ -504,16 +505,7 @@ def default_urlconf(request):
     with Path(CURRENT_DIR, 'templates', 'default_urlconf.html').open() as fh:
         t = DEBUG_ENGINE.from_string(fh.read())
     c = Context({
-        "title": _("Welcome to Django"),
-        "heading": _("It worked!"),
-        "subheading": _("Congratulations on your first Django-powered page."),
-        "instructions": _(
-            "Next, start your first app by running <code>python manage.py startapp [app_label]</code>."
-        ),
-        "explanation": _(
-            "You're seeing this message because you have <code>DEBUG = True</code> in your "
-            "Django settings file and you haven't configured any URLs. Get to work!"
-        ),
```

Comment:
It would be nice to keep the page translatable. I'm not sure the strings need to be translated in Python, perhaps the template could `{% load i18n %}` and use the regular template translation tooling, with appropriate tweaks to the `DEBUG_ENGINE` configuration.